packages/spark-core/components/_buttons.scss

```diff
@@ -80,12 +80,12 @@
   &:active::after,
   &:focus::after {
     border-color: $btn-tertiary-hover-line-color;
-    animation: width-zero-to-full $btn-transition-speed ease-in-out;
```

Comment:
With this change, do we still need this animation code? Is it something that we can go and also remove?

Reply:
width-zero-to-full is still used in a couple places.